test/unit/inet-address-tests.js

```diff
@@ -168,28 +168,28 @@ describe("InetAddress", function () {
             ].forEach(function (address) {
                 assert.throws(function () {
                     InetAddress.fromString(address);
-                }, TypeError);
+                }, Error);
             });
         });
         it("should throw TypeError when the string is not a valid address", function () {
             assert.throws(function () {
                 InetAddress.fromString("127.0.0.1.10");
-            }, TypeError);
+            }, Error);
             assert.throws(function () {
                 InetAddress.fromString("127.0.");
-            }, TypeError);
+            }, Error);
             assert.throws(function () {
                 InetAddress.fromString(" 127.10.0.1");
-            }, TypeError);
+            }, Error);
             assert.throws(function () {
                 InetAddress.fromString("z");
-            }, TypeError);
+            }, Error);
             assert.throws(function () {
                 InetAddress.fromString("::Z:11:2233:4455:aa:bb");
-            }, TypeError);
+            }, Error);
             assert.throws(function () {
                 InetAddress.fromString(" ::a:11");
-            }, TypeError);
+            }, Error);
```

Comment:
Maybe put this value to array and use `forEach`?

Reply:
Do u mean something like this:
```rust
const testCases = [
                { buffer: utils.allocBufferUnsafe(10), shouldThrow: true },
                { buffer: null, shouldThrow: true },
                { buffer: undefined, shouldThrow: true },
                { buffer: utils.allocBufferUnsafe(16), shouldThrow: false },
                { buffer: utils.allocBufferUnsafe(4), shouldThrow: false }
            ];

            testCases.forEach(({ buffer, shouldThrow }) => {
                if (shouldThrow) {
                    assert.throws(() => new InetAddress(buffer), Error);
                } else {
                    assert.doesNotThrow(() => new InetAddress(buffer), Error);
                }
            });
```

Reply:
Yes :>